routes/gallery.js

```diff
@@ -0,0 +1,106 @@
+const express = require("express");
+const router = express.Router();
+const Gallery = require("../database/models/Gallery");
+
+function isAuthenticated(req, res, next) {
+  if (req.isAuthenticated()) {
+    return next();
+  } else {
+    return res.redirect("/login.html");
+  }
+}
+
+router.get("/new", isAuthenticated, (req, res) => {
+  res.render("newPhoto");
+});
+
+router.get("/:id", (req, res) => {
+  Gallery.where({ id: req.params.id })
+    .fetchAll({ withRelated: ["user"] })
```

Comment:
Prefer fetch instead of fetchAll in this case, since you're only getting a single gallery